#! /bin/sh /usr/share/dpatch/dpatch-run
## originally: 08_default_units.dpatch by Tilman Koschnick <til@subnetz.org>
## now default_units.dpatch, updated by Bernd Zeimetz <bernd@bzed.de>
##
## DP: Use metric units by default.

@DPATCH@
diff -urNad gpsd~/xgps.ad gpsd/xgps.ad
--- gpsd~/xgps.ad	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/xgps.ad	2007-10-14 15:03:10.000000000 +0200
@@ -10,5 +10,5 @@
 *climb*labelString:      Climb
 *status*labelString: Status   
 !*quit*labelString:       Quit     
-*speedunits:             mph
-*altunits:               feet
+*speedunits:             kph
+*altunits:               meters
diff -urNad gpsd~/xgpsspeed.ad gpsd/xgpsspeed.ad
--- gpsd~/xgpsspeed.ad	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/xgpsspeed.ad	2007-10-14 15:02:34.000000000 +0200
@@ -15,5 +15,5 @@
 *meter*width:		120
 *meter*height:		120
 *needleColor:		salmon1
-*speedunits:		mph
-*altunits:		feet
+*speedunits:		kph
+*altunits:		meters
